lib/pages/home/home_controller.dart

```diff
@@ -1,15 +1,31 @@
 import 'package:get/get.dart';
+import 'package:intl/intl.dart';
 import 'package:survey/managers/user_manager.dart';
 
+const HOME_DISPLAY_DATE_FORMAT = 'EEEE, MMMM dd';
+
 class HomeController extends GetxController {
+  final currentDateText = "".obs;
+  final currentUserAvatar = "".obs;
```

Comment:
I think we should not expose the HomeController's mutable value outside. Imo we should create immutable values and listen the change through it 🙏